packages/grafana-data/src/field/fieldOverrides.ts

```diff
@@ -290,10 +290,42 @@ export function setDynamicConfigValue(config: FieldConfig, value: DynamicConfigV
   }
 }
 
+/**
+ * Merge the links from the ds config and the panel config
+ * @param configLinks coming from the datasource
+ * @param defaultLinks coming from the panel config
+ * @returns unique merged links
+ */
+
+function mergeLinks(configLinks: DataLink[], defaultLinks: DataLink[]) {
```

Comment:
```suggestion
function mergeLinks(dataSourceLinks: DataLink[], defaultLinks: DataLink[]) {
```

Reply:
Please apply this feedback to all places where data source provided links are named `config links`